docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-defining-a-stage-proc.adoc

```diff
@@ -1,2 +1,28 @@
-[id='']
-= TODO
\ No newline at end of file
+[id='case-management-defining-a-stage-proc']
+= Defining a stage
+
+A stage can be modeled in BPMN2 using the case designer. Stages are a way of grouping related tasks in a way that clearly define activities that must complete before the next stage of the case commences. For example, the IT Orders case definition described in <<case-management-designing-a-case-proc>> can also be defined using stages in the following way:
+
+.IT Orders project stages example
+image::itorders-stages.png[IT Orders - stages]
+
+.Prerequisites
+
+* You have followed the procedure in <<case-management-creating-a-case-proc>> to create a new case in {CENTRAL}.
+* You have started a case definition and defined any other required case nodes described in <<case-management-designing-a-case-proc>>. Stages must be triggered by another node within a case definition, therefore at least one other node must be configured in the designer before the stage can be started.  
```

Comment:
Technically, either "definition; therefore, at least" or just "definition, so at least". I vote for the latter, simpler.

Reply:
ack.